unlock-app/src/components/content/SettingsContent.tsx

```diff
@@ -54,15 +58,20 @@ export class SettingsContent extends React.Component<
       this.setState({
         stripe: window.Stripe(stripeApiKey),
       })
-      if (this.interval) {
-        clearInterval(this.interval)
-      }
+      this.removeInterval()
+    }
+  }
+
+  removeInterval() {
```

Comment:
This was just a bit of cleanup to make sure we get rid of the interval if needed.